src/App.jsx

```diff
@@ -1,14 +1,29 @@
-import reactLogo from './assets/react.svg'
 import './App.css'
+import Header from './components/Header/Header'
+import Courses from './components/Courses/Courses'
+import CreateCourse from './components/CreateCourse/CreateCourse'
+import { authorsContext, courseListContext, isTogglePage } from './helpers/context'
+import { useState } from 'react'
+import { mockedAuthorsList, mockedCoursesList } from './constants'
 
 function App() {
+  const [allAuthors, setAllAuthors] = useState(mockedAuthorsList)
+  const [CourseList, setCourseList] = useState(mockedCoursesList)
```

Comment:
please use `courseList` instead

Reply:
In this task, the lesson didn't give me an API for getting data, so I must use mockCourseList to mock it, I will remove it in task3